subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -114,6 +114,7 @@ struct lwm2m_rd_client_info {
 	lwm2m_ctx_event_cb_t event_cb;
 
 	bool use_bootstrap : 1;
+
```

Comment:
unnecessary whitespace change.

Reply:
Uuh There is a second one also. Good spot